marl_eval/utils/data_processing_utils.py

```diff
@@ -47,6 +47,29 @@ def check_absolute_metric(steps: List) -> Union[str, None]:
     return None
 
 
+def check_comma_in_algo_names(
+    algos: List, get_valid: bool = False
+) -> Tuple[bool, list]:
+    """Check that no algorithm names contain commas.
+
+    Args:
+        algos: list containing names of algorithms in experiment.
+        get_valid: boolean dictating whether valid or invalid
+            algorithm names should be returned. If `True` the names
+            of valid algorithms will be returned and if `False` the
+            names of invalid algorithms will be returned.
+    """
+    comma_in_names = [("," not in str(name)) for name in algos]
```

Comment:
Do you think only comma is the non-valid format maybe ":" ";" too... so we can add them all here (we can make the idea to check str include only characters, digits, and '_' or '-')

Reply:
All other punctuation will be fine. The error happens because of lines 260-264 [here](https://github.com/instadeepai/marl-eval/blob/aa3b5df13e56fad263ba0a3151f5eb452abe5d22/marl_eval/plotting_tools/plotting.py#L260). That is why we have to explicitly check for it :sweat_smile: